app.py

```diff
@@ -1,12 +1,34 @@
-from flask import Flask, render_template, request, redirect, url_for
+from flask import Flask, render_template, request, redirect, url_for, jsonify
 import session_items as session
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', title='Home',items=session.get_items())
 
+@app.route('/todo', methods=['POST'])
+def new_todo():
+    data = request.json
+    print("data is " + format(data))
+    session.add_item(data['title'])
+    return jsonify(success=True)
+    
+@app.route('/todo/<item_id>', methods=['PUT'])
+def update_todo(item_id):
+    data = request.json
+    print("data is " + format(data))
+    item = session.get_item(item_id)
+    item['status'] = data['status']
+    session.save_item(item)
```

Comment:
Should probably check if the todo name is empty here (i.e. add some server side validation)